.devcontainer/devcontainer.json

```diff
@@ -4,7 +4,14 @@
   "workspaceFolder": "/workspace",
   "service": "app",
   "forwardPorts": [1234, 8080, 4000, 9000],
-  "settings": {},
-  "extensions": [],
+  "customizations": {
```

Comment:
Is it necessary?

Reply:
This is the newer format and also makes sure that autoformatting is working correctly in the dev container without modifications or user settings.